notebooks/submission_tutorial/2_behind_the_scenes.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>Let's changed "show" -&gt; "demo". Or somehow make it clear that "to demo this we will use the pre-training dataset for which the test responses are available".</p><p>It's probably due to my stupidity, but when I read this first I was wondering what does the pre-training dataset have to do with the competition, and then I realized it's just an example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sinzlab/sensorium/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>